app/models/application.rb

```diff
@@ -0,0 +1,171 @@
+# == Schema Information
+#
+# Table name: applications
+#
+#  id                     :integer          not null, primary key
```

Comment:
Just for now. It's in my "to do" section of the overview for this PR to create an event model. You're right - it'll be

User has many Applications
Event has many Applications
Application belongs to User
Application belongs to Event

But right now I don't have an event model, so I'm skipping that for now.